src/zeit/arbeit/interfaces.py

```diff
@@ -33,3 +36,17 @@ class IZARCenterPage(
         zeit.content.cp.interfaces.ICenterPage,
         zeit.cms.section.interfaces.ISectionMarker):
     pass
+
+
+class IZARInfobox(
+        zeit.content.infobox.interfaces.IInfobox,
+        zeit.cms.section.interfaces.ISectionMarker):
+    pass
+
+
+class IDebate(zope.interface.Interface):
```

Comment:
Das ist jetzt ja genau das gleiche wie in zeit.campus... dann würde ich das IDebate und seine Implementierung glaub ich nach zeit.content.infobox hochziehen, und in campus+arbeit benutzen.